prow/config/tide.go

```diff
@@ -292,7 +292,25 @@ func (qm *QueryMap) ForRepo(org, repo string) TideQueries {
 // * a label that is in both the labels and missing_labels section
 // * a branch that is in both included and excluded branch set.
 func (tq *TideQuery) Validate() error {
-	orgs, repos, excludedRepos := sets.NewString(), sets.NewString(), sets.NewString()
+	duplicates := func(field string, list []string) error {
+		dups := sets.NewString()
+		dupCount := 0
+		seen := sets.NewString()
+		for _, elem := range list {
+			if seen.Has(elem) {
+				dupCount++
+				dups.Insert(elem)
+			} else {
+				seen.Insert(elem)
+			}
+		}
+		if dupCount == 0 {
```

Comment:
Technically you could just use `len(list) - len(seen.UnorderedList())` instead of tracking the number of dupes

Reply:
:man_facepalming: Fixed.